core/build.gradle

```diff
@@ -19,8 +19,8 @@ repositories {
 }
 
 dependencies {
-    antlr "org.antlr:antlr4:4.9.3"
-    compileOnly "org.antlr:antlr4-runtime:4.9.3"
+    antlr "org.antlr:antlr4:4.13.1"
+    compileOnly "org.antlr:antlr4-runtime:4.13.1"
 
```

Comment:
I ran into a bunch of errors until I did a full gradle dependency refresh in Intellij:


https://github.com/broadinstitute/juniper/assets/17440010/7ffde7bd-5a9b-418c-af14-2f71b43f7efb